client/src/config.rs

```diff
@@ -90,6 +91,7 @@ impl Config {
 
 #[cfg(test)]
 mod tests {
+    use assertables::{assert_contains, assert_contains_as_result};
```

Comment:
Unused import?

Reply:
It's used by the output of `assert_contains`. Maybe it should be patched, actually... Out of scope of this PR anyway.